src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045.enums.dfdl.xsd

```diff
@@ -0,0 +1,1067 @@
+      <xs:enumeration value="UNITED_STATES_(US)" dfdlx:repValues="248"/>
+      <xs:enumeration value="URUGUAY_(UY)" dfdlx:repValues="249"/>
+      <xs:enumeration value="UZBEKISTAN_(UZ)" dfdlx:repValues="250"/>
+      <xs:enumeration value="VANUATU_(NH)" dfdlx:repValues="251"/>
+      <xs:enumeration value="VATICAN_CITY_(VT)" dfdlx:repValues="252"/>
+      <xs:enumeration value="VENEZUELA_(VE)" dfdlx:repValues="253"/>
+      <xs:enumeration value="VIETNAM_(VM)" dfdlx:repValues="254"/>
+      <xs:enumeration value="VIRGIN_ISLANDS,_US_(VQ)" dfdlx:repValues="255"/>
+      <xs:enumeration value="WAKE_ISLAND_(WQ)" dfdlx:repValues="256"/>
+      <xs:enumeration value="WALLIS_AND_FUTUNA_(WF)" dfdlx:repValues="257"/>
+      <xs:enumeration value="WEST_BANK_(WE)" dfdlx:repValues="258"/>
+      <xs:enumeration value="WESTERN_SAHARA_(WI)" dfdlx:repValues="259"/>
+      <xs:enumeration value="YEMEN_(YM)" dfdlx:repValues="260"/>
+      <xs:enumeration value="YUGOSLAVIA_(YI)" dfdlx:repValues="261"/>
+      <xs:enumeration value="ZAMBIA_(ZA)" dfdlx:repValues="262"/>
+      <xs:enumeration value="ZIMBABWE_(ZI)" dfdlx:repValues="263"/>
+      <xs:enumeration value="EXERCISE_BLACK_COUNTRY_(OA)" dfdlx:repValues="264"/>
+      <xs:enumeration value="EXERCISE_BLACK_FORCES_(OB)" dfdlx:repValues="265"/>
+      <xs:enumeration value="EXERCISE_BLUE_COUNTRY_(OC)" dfdlx:repValues="266"/>
+      <xs:enumeration value="EXERCISE_BLUE_FORCE_(OD)" dfdlx:repValues="267"/>
+      <xs:enumeration value="EXERCISE_FRIENDLY_COUNTRY_(YC)" dfdlx:repValues="268"/>
+      <xs:enumeration value="EXERCISE_FRIENDLY_FORCE_(YY)" dfdlx:repValues="269"/>
+      <xs:enumeration value="EXERCISE_HOSTILE_COUNTRY_(XC)" dfdlx:repValues="270"/>
+      <xs:enumeration value="EXERCISE_HOSTILE_FORCE_(XX)" dfdlx:repValues="271"/>
+      <xs:enumeration value="EXERCISE_NEUTRAL_COUNTRY_(ZC)" dfdlx:repValues="272"/>
+      <xs:enumeration value="EXERCISE_NEUTRAL_FORCE_(ZZ)" dfdlx:repValues="273"/>
+      <xs:enumeration value="EXERCISE_ORANGE_COUNTRY_(OJ)" dfdlx:repValues="274"/>
+      <xs:enumeration value="EXERCISE_ORANGE_FORCE_(OK)" dfdlx:repValues="275"/>
+      <xs:enumeration value="EXERCISE_RED_COUNTRY_(OR)" dfdlx:repValues="276"/>
+      <xs:enumeration value="EXERCISE_RED_FORCE_(OE)" dfdlx:repValues="277"/>
+      <xs:enumeration value="EXERCISE_WHITE_COUNTRY_(ON)" dfdlx:repValues="278"/>
+      <xs:enumeration value="EXERCISE_NATO_FORCE_(OT)" dfdlx:repValues="279"/>
+      <xs:enumeration value="EXERCISE_PURPLE_FORCE_(OL)" dfdlx:repValues="280"/>
+      <xs:enumeration value="EXERCISE_SPARE_NUMBER_ONE_(XA)" dfdlx:repValues="281"/>
+      <xs:enumeration value="EXERCISE_SPARE_NUMBER_TWO_(XB)" dfdlx:repValues="282"/>
+      <xs:enumeration value="EXERCISE_UNITED_NATIONS_FORCE_(UU)" dfdlx:repValues="283"/>
+      <xs:enumeration value="EXERCISE_FORMER_WARSAW_PACT_FORCE_(OW)" dfdlx:repValues="284"/>
+      <xs:enumeration value="SPANISH_NORTH_AFRICA_(SQ)" dfdlx:repValues="313"/>
```

Comment:
285 to 312 are missing, but I just fixed this bug in VMF, so the below are the missing values

```
      <xs:enumeration value="NAT/ALL-1" dfdlx:repValues="285"/>
      <xs:enumeration value="NAT/ALL-2" dfdlx:repValues="286"/>
      <xs:enumeration value="NAT/ALL-3" dfdlx:repValues="287"/>
      <xs:enumeration value="NAT/ALL-4" dfdlx:repValues="288"/>
      <xs:enumeration value="NAT/ALL-5" dfdlx:repValues="289"/>
      <xs:enumeration value="NAT/ALL-6" dfdlx:repValues="290"/>
      <xs:enumeration value="NAT/ALL-7" dfdlx:repValues="291"/>
      <xs:enumeration value="NAT/ALL-8" dfdlx:repValues="292"/>
      <xs:enumeration value="NAT/ALL-9" dfdlx:repValues="293"/>
      <xs:enumeration value="NAT/ALL-10" dfdlx:repValues="294"/>
      <xs:enumeration value="NAT/ALL-11" dfdlx:repValues="295"/>
      <xs:enumeration value="NAT/ALL-12" dfdlx:repValues="296"/>
      <xs:enumeration value="NAT/ALL-13" dfdlx:repValues="297"/>
      <xs:enumeration value="NAT/ALL-14" dfdlx:repValues="298"/>
      <xs:enumeration value="NAT/ALL-15" dfdlx:repValues="299"/>
      <xs:enumeration value="NAT/ALL-16" dfdlx:repValues="300"/>
      <xs:enumeration value="NAT/ALL-17" dfdlx:repValues="301"/>
      <xs:enumeration value="NAT/ALL-18" dfdlx:repValues="302"/>
      <xs:enumeration value="NAT/ALL-19" dfdlx:repValues="303"/>
      <xs:enumeration value="NAT/ALL-20" dfdlx:repValues="304"/>
      <xs:enumeration value="NAT/ALL-21" dfdlx:repValues="305"/>
      <xs:enumeration value="NAT/ALL-22" dfdlx:repValues="306"/>
      <xs:enumeration value="NAT/ALL-23" dfdlx:repValues="307"/>
      <xs:enumeration value="NAT/ALL-24" dfdlx:repValues="308"/>
      <xs:enumeration value="NAT/ALL-25" dfdlx:repValues="309"/>
      <xs:enumeration value="NAT/ALL-26" dfdlx:repValues="310"/>
      <xs:enumeration value="NAT/ALL-27" dfdlx:repValues="311"/>
      <xs:enumeration value="NAT/ALL-28" dfdlx:repValues="312"/>
```